src/lua/compat.lua

```diff
@@ -127,6 +127,16 @@ more verbose representation is used.
 https://tarantool.io/compat/box_error_serialize_verbose
 ]]
 
+local BOX_CONSIDER_SYSTEM_SPACES_SYNCHRONOUS = [[
+Controls whether to consider system spaces indefinitely synchronous when the
```

Comment:
nit: it's not clear what "indefinitely synchronous" means

Reply:
Fixed.